webapp/packages/plugin-resource-manager/src/ResourceManagerService.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2022 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+
+import { makeObservable, observable } from 'mobx';
+
+import { injectable } from '@cloudbeaver/core-di';
+import { GraphQLService } from '@cloudbeaver/core-sdk';
+
+export const RESOURCE_MANAGER_RESOURCE_NODE_TYPE = 'rm.resource';
+export const RESOURCE_MANAGER_PROJECT_NODE_TYPE = 'rm.project';
+
+@injectable()
+export class ResourceManagerService {
```

Comment:
move to `ResourceManagerResource` (lol)
`createResource`, `writeResource`, `readResource`, `deleteResource`